packages/story-editor/src/components/library/panes/media/common/innerElement.js

```diff
@@ -252,6 +254,7 @@ InnerElement.propTypes = {
   height: PropTypes.number,
   isMuted: PropTypes.bool,
   onClick: PropTypes.func.isRequired,
+  onLoad: PropTypes.func,
```

Comment:
```suggestion
  onLoad: PropTypes.func.isRequired,
```

Otherwise, if not required you should do `onLoad = noop` above

Reply:
Fixed https://github.com/google/web-stories-wp/pull/9688/commits/56d22933332e4d7d2b05e4644b908316216181c6